components/IconButtonWithLabel.js

```diff
@@ -27,7 +27,7 @@ export default function IconButtonWithLabel({ onPress, label, icon }) {
   return (
     <View style={styles.container}>
       <View style={styles.label}>
-        <Caption color={theme.colors.surface}>{label}</Caption>
+        <Caption style={{ color: theme.colors.surface }}>{label}</Caption>
```

Comment:
why this change? I would prefer no inline styles and exposing a color property from our typography components which allows overriding whatever default value

Reply:
Fixed https://github.com/nearform/optic-expo/pull/28/commits/595446b3e7eb98d72fa6c3cd102872177de942cb.